src/Filtering/BinaryMathematicalMorphology/ClosingBinaryImage/Code.cxx

```diff
@@ -32,32 +33,35 @@ using ImageType = itk::Image<unsigned char, 2>;
 
 static void
 CreateImage(ImageType * const image);
+static void
+WriteImage(const ImageType::Pointer image, const std::string & fileName);
```

Comment:
This function was recently integrated into ITK. It is declared in `itkImageFileWriter.h`, see https://github.com/InsightSoftwareConsortium/ITK/commit/d8af2086421caabb8f7a47586e7acdc714ceb534.